ceph-ansible.spec.in

```diff
@@ -55,8 +55,7 @@ pushd %{buildroot}%{_datarootdir}/ceph-ansible
 popd
 
 %check
-# Borrowed from upstream's .travis.yml:
-ansible-playbook -i dummy-ansible-hosts test.yml --syntax-check
+ansible-playbook -i localhost, test.yml --syntax-check
```

Comment:
The syntax `localhost,` with the comma is a bit misleading even though it's correct. Perhaps we should just add `--connection=local` to the CLI and remove `-i localhost,`?

Thanks.